### PR TITLE
BXMSDOC-4191-master: Update managing and monitoring execution server doc per changes in 7.4

### PR DESCRIPTION
**Pull request by kaldesai**
[Epic](https://issues.jboss.org/browse/BXMSDOC-4001)
[JIRA](https://issues.jboss.org/browse/BXMSDOC-4191)
[managing and monitoring process server doc preview](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4191-PAM-NoteAdded/#migration-configure-kie-server-proc)
[managing and monitoring decision server doc preview](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4191-DM-NoteAdded/#migration-configure-kie-server-proc)

Please review and verify the **note** present at the bottom of chapter 4. Configuring and starting Process Server. 
Thank you

### CODE REVIEW

File: doc-content/enterprise-only/installation/eap-data-source-add-proc.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ For production environments, specify an actual data source. Do not use the examp
 
 .Prerequisites
 * The JDBC providers that you want to use to create database connections are configured on all servers on which you want to deploy {KIE_SERVER}, as described in the "Creating Datasources" and "JDBC Drivers" sections of the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html/configuration_guide/index[_Red Hat JBoss Enterprise Application Server Configuration Guide_].
-* The *{PRODUCT} {PRODUCT_VERSION_LONG} Add Ons* (`{PRODUCT_FILE}-add-ons.zip`) file has been downloaded, as described in <<install-download-proc_{context}>>.
+* The {PRODUCT} {PRODUCT_VERSION_LONG} Add Ons ({PRODUCT_FILE}-add-ons.zip) file has been downloaded from the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal.
```
Comment (pnagesha) [inline doc-content/enterprise-only/installation/eap-data-source-add-proc.adoc]: Remove "has been". See style guide for more information on tense.

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc [hunk 2]
```diff
@@ -58,3 +58,32 @@ If the {CENTRAL} port is 8080, as in this example, then the {KIE_SERVER} port, w
 ====
 
 {KIE_SERVER} connects to the new {CENTRAL} and collects the list of deployment units (KIE containers) to be deployed.
+
+ifdef::PAM[]
+[NOTE]
+====
+When the class is used inside a dependency JAR file to access {KIE_SERVER} from {KIE_SERVER} client, you get the `ConversionException` and `ForbiddenClassException` in {CENTRAL}.
```
Comment (pnagesha) [inline doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: When you use a class inside....

Use active voice.

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc [hunk 3]
```diff
@@ -58,3 +58,32 @@ If the {CENTRAL} port is 8080, as in this example, then the {KIE_SERVER} port, w
...
+To avoid generating these exceptions in {CENTRAL}, do any of the following tasks:
```
Comment (pnagesha) [inline doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: do one of the following:

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc [hunk 4]
```diff
@@ -58,3 +58,32 @@ If the {CENTRAL} port is 8080, as in this example, then the {KIE_SERVER} port, w
...
+
+* If the exceptions are generated at client side, add following system property to the kie-server client:
```
Comment (pnagesha) [inline doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: .... on the client-side

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc [hunk 5]
```diff
@@ -58,3 +58,32 @@ If the {CENTRAL} port is 8080, as in this example, then the {KIE_SERVER} port, w
...
+[source]
+----
+System.setProperty("org.kie.server.xstream.enabled.packages", "org.example.**");
+----
+* If the exceptions are generated at server side, go to {PRODUCT} installation directory, open `standalone-full.xml` and under the <system-properties> tag, set the following property:
```
Comment (pnagesha) [inline doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: ... on the server-side

Comment (pnagesha) [inline doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: open 'standalone-full.xml' from the {Product} installation directory. Set the following property under the <system-properties> tag.

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc [hunk 6]
```diff
@@ -58,3 +58,32 @@ If the {CENTRAL} port is 8080, as in this example, then the {KIE_SERVER} port, w
...
+----
+<property name="org.kie.server.xstream.enabled.packages" value="org.example.**"/>
+----
+* Set the following JVM property:
+[source]
+----
+-Dorg.kie.server.xstream.enabled.packages=org.example.**
+----
+
+The `org.example` package is mandatory. You can specify multiple packages separated by comma , for example,
```
Comment (pnagesha) [inline doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: What do you mean here? Should the user specify the package name as org.example?

File: doc-content/enterprise-only/installation/eap-data-source-add-proc.adoc (hunk 1 above)
Comment (kaldesai) [reply doc-content/enterprise-only/installation/eap-data-source-add-proc.adoc]: Done

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc (hunk 2 above)
Comment (kaldesai) [reply doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: Done

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc (hunk 4 above)
Comment (kaldesai) [reply doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: Ack

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc (hunk 5 above)
Comment (kaldesai) [reply doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: Ack

Comment (kaldesai) [reply doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: Ack

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc (hunk 6 above)
Comment (kaldesai) [reply doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: Yes, org.example is the main package name.

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc [hunk 7]
```diff
@@ -58,3 +58,32 @@ If the {CENTRAL} port is 8080, as in this example, then the {KIE_SERVER} port, w
...
+[source]
+----
+-Dorg.kie.server.xstream.enabled.packages=org.example.**
+----
+It is expected that you do not configure the classes that exits in KJAR using these system property. Ensure that only known classes are used in the system property to avoid any vulnerabilities.
+
+The `org.example` package is mandatory. You can specify multiple packages separated by comma , for example,
+`org.example1.* * , org.example2.* * , org.example3.* *`.
+
+You can also add specific classes , for example, `org.example1.* * , org.example2.Mydata2`.
```
Comment (MarianMacik) [inline doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: Should be perhaps `org.example1.Mydata1`?

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc (hunk 6 above)
Comment (MarianMacik) [reply doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: I don't think it is mandatory, if a user doesn't have this package in his project, is it?

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc [hunk 8]
```diff
@@ -58,3 +58,32 @@ If the {CENTRAL} port is 8080, as in this example, then the {KIE_SERVER} port, w
...
+[source]
+----
+<property name="org.kie.server.xstream.enabled.packages" value="org.example.**"/>
+----
+* Set the following JVM property:
+[source]
+----
+-Dorg.kie.server.xstream.enabled.packages=org.example.**
+----
+It is expected that you do not configure the classes that exits in KJAR using these system property. Ensure that only known classes are used in the system property to avoid any vulnerabilities.
```
Comment (MarianMacik) [inline doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: Should be `...classes that exists...`.

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc (hunk 6 above)
Comment (sutaakar) [reply doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: org.example is not a mandatory package, it is just an example. User should define any package he wants.

Comment (kaldesai) [reply doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: Ack.

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc (hunk 8 above)
Comment (kaldesai) [reply doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: Done.

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc (hunk 7 above)
Comment (kaldesai) [reply doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: Done

**Pull request merged**